### ISSUE TITLE
Scary behavior when disk full

### ISSUE THREAD

**Issue by mheder** [standardnotes/forum#86]
With desktop version 1.2.4, under linux I had today a somewhat unpleasant experience. 

After booting up my operating system I started Standard Notes and it showed only two entries, both from today. This was unsettling as in reality I have about 250 notes. The account -> download data archive also only showed two entries in the JSON. I grabbed my phone and I saw all my notes, also I went to the data dashboard and again all my notes were there. This is when I figured out that my disk was full. This must have caused the problem. 

The only thing I could do was signing out and delete all the data, and then signing in again (after freeing up space of course). 

I believe it could be good to have some warning message when the disk is full and not show an inconsistent state of local data.

**Comment by mobitar** [comment]
Thanks for the report. That's an interesting edge case. I'm not sure if we can somehow detect low storage, but I'll look into this.

**Comment by mobitar** [comment]
Yeah, I would have expected that too. However, when you're in a state of low storage, it's sometimes hard to predict how the OS will behave. It's not something we've explicitly coded for either.

**Comment by chanclr** [comment]
I got this with Windows version 3.3.5. Definitely a shock, but luckily I was able to sync my notes back. For what it's worth, I was able to recreate it with a new account filled with gibberish data.

**Comment by mobitar** [comment]
Did you get an error alert at all? This should be something our app notifies you about. If not, this is something that needs to be remedied.

cc @baptiste-grob

**Comment by chanclr** [comment]
It was the potentially out of sync indicator at the bottom right, but after notes have been lost. It displayed the 3 options of restart app, sign out, or sync resolution.

**Comment by baptiste-grob** [comment]
We definitely need to show something more explicit then. When your disk was considered full, do you remember what Windows was reporting? Did you still have a couple of megabytes left, for example. Did any external warnings pop up?

**Comment by chanclr** [comment]
Windows had a once-off pop-up saying that disk space was nearly full and asking me if I wanted to clean up space. This was I believe when there was still a few hundred megabytes left, which I dismissed. The first time I was downloading something, which was what filled the disk in the first place. I think it got to 0 bytes left in This PC. The second time I was constantly on purpose filling the disk to try and keep it at 0 bytes left (Windows seems to clear up space on its own by deleting something).
